bin/start_server.sh

```diff
@@ -12,9 +12,11 @@ cp -n /usr/src/app/ansible/roles/screenly/files/default_assets.yml /data/.screen
 
 echo "Running migration..."
 
-./manage.py initialize_assets
-./manage.py makemigrations
-./manage.py migrate --fake-initial
+# The following block ensures that the migration is transactional and that the
+# database is not left in an inconsistent state if the migration fails.
+cp /data/.screenly/screenly.db /data/.screenly/backup.db
```

Comment:
Above, you do `mkdir -p ... /data/.screenly ...`, which will make an empty directory.
But here, you expect the existence of a screenly db.

Could you rewrite the script so it will make sense?

Reply:
@renatgalimov, I accidentally deleted the line that copies `screenly.db` from the `./ansible` folder.
I'll bring it back. I hope it makes more sense later.

Reply:
Now, it makes even less sense to me🤔 Is the DB expected to be modified? Would I lose my modifications every time I restart this script?